packages/common/src/ui/layout/tables/types.ts

```diff
@@ -44,19 +44,13 @@ export interface DataTableApi<T> {
   selectedRows: T[];
 }
 
-interface Pagination {
-  first: number;
-  offset: number;
-  total: number;
-}
-
 export interface TableProps<T extends Record<string, unknown>> {
   columns: Column<T>[];
   data?: T[];
   sortBy: SortBy<T>;
   isLoading?: boolean;
   onSortBy: (sortRule: SortRule<T>) => void;
-  pagination: Pagination;
+  pagination: { page: number; total?: number; offset: number; first: number };
```

Comment:
to avoid confusion with the `Pagination` interface exported from `usePagination`?

Reply:
hmm also that `Pagination` has callbacks, changed em a little here 
66b3994 and 52660ed